tests/test_loaders.py

```diff
@@ -9,8 +9,7 @@
 from unicef_vision.loaders import FileDataLoader, ManualDataLoader, VISION_NO_DATA_MESSAGE, VisionDataLoader
 
 FAUX_VISION_URL = 'https://api.example.com/foo.svc/'
-FAUX_VISION_USER = 'jane_user'
-FAUX_VISION_PASSWORD = 'password123'
+FAUX_INSIGHT_SUB_KEY = 'insight_sub_key'
```

Comment:
Why do we have this faux sub key? It appears to be the same as the normal sub key